Airdrop.test.js

```diff
@@ -0,0 +1,136 @@
+const {
+    BN,
+    send,
+    constants,
+    expectRevert,
+    assertRevert,
+    balance,
+    ether
+} = require('@openzeppelin/test-helpers');
+require('chai')
+    .use(require('chai-as-promised'))
+    .use(require('chai-bn')(BN))
+    .should();
+
+const Token = artifacts.require("Token");
+const Airdrop = artifacts.require("Airdrop");
+
+contract("Airdrop", ([owner, user1, user2, user3]) => {
+
+    beforeEach(async () => {
+        tokenInstance = await Token.new({ from: owner });
+        airdropInstance = await Airdrop.new(tokenInstance.address, { from: owner });
+    });
+

```

Comment:
use the following order in the tests:
describe("Function name", async() =>{
 it("", ...
 it("",...
}